staging/src/k8s.io/apiserver/pkg/audit/metrics.go

```diff
@@ -81,7 +81,7 @@ func HandlePluginError(plugin string, err error, impacted ...*auditinternal.Even
 	msg := fmt.Sprintf("Error in audit plugin '%s' affecting %d audit events: %v\nImpacted events:\n",
 		plugin, len(impacted), err)
 	for _, ev := range impacted {
-		msg = msg + EventString(ev) + "\n"
+		msg = msg + EventString(ev)
```

Comment:
Ah, I see. Maybe we should remove the newline from EventString instead then? I think that would be more clear.